src/Illuminate/Redis/Connections/PhpRedisConnection.php

```diff
@@ -195,9 +195,14 @@ public function brpop(...$arguments)
      * @param  int|null  $count
      * @return mixed|false
      */
-    public function spop($key, $count = 1)
+    public function spop($key, $count = null)
```

Comment:
This is a breaking change to the method signature

Reply:
use func_get_args() instead of [$key, $count], and add a comment for the reason.

may be we can deprecated this method and remove in the future.